src/terrain/routes/schemas/admin.clj

```diff
@@ -1,22 +1,30 @@
 (ns terrain.routes.schemas.admin
   (:use [common-swagger-api.schema
          :only [describe
+                NonBlankString
                 SortFieldDocs
                 SortFieldOptionalKey]]
         [schema.core :only [defschema enum]])
-  (:require [common-swagger-api.schema.apps.admin.apps :as apps-schema]))
+  (:require [common-swagger-api.schema.apps :as apps-schema]
+            [common-swagger-api.schema.apps.admin.apps :as admin-apps-schema]))
 
-;; Convert the keywords in AdminAppSearchValidSortFields to strings,
-;; so that the correct param format is passed through to the apps service.
+;; Convert Date params and keywords in enum values to strings,
+;; so that the correct param formats are passed through to the apps service.
 (defschema AdminAppSearchParams
-  (merge apps-schema/AdminAppSearchParams
-         {SortFieldOptionalKey
-          (describe (apply enum (map name apps-schema/AdminAppSearchValidSortFields))
+  (merge admin-apps-schema/AdminAppSearchParams
+         {apps-schema/AppJobStatsStartDateOptionalParam
+          (describe NonBlankString apps-schema/AppJobStatsStartDateParamDocs)
+
+          apps-schema/AppJobStatsEndDateOptionalParam
+          (describe NonBlankString apps-schema/AppJobStatsEndDateParamDocs)
```

Comment:
Unfortunately, changing these query params from `Date` to `NonBlankString` params displays their `Data Type` in terrain's Swagger UI as `string` rather than `date-time` types, and our `doc-only` schema function will not work with `query` schemas.